src/test/skript/tests/syntaxes/expressions/ExprLowestHighestSolidBlock.sk

```diff
@@ -0,0 +1,36 @@
+test "lowest/highest solid block (old height)" when running below minecraft "1.18":
+
+	# highest solid block
+	set block at location(0.5, 255.5, 0.5, "world") to air
+	set block at location(0.5, 254.5, 0.5, "world") to air
+	set block at location(0.5, 253.5, 0.5, "world") to dirt
+	set {_highest} to highest solid block at location(0.5, 64, 0.5, "world")
+	assert type of {_highest} is dirt with "highest block is not dirt (got '%type of {_highest}%')"
+	assert location of {_highest} is location(0.5, 253.5, 0.5, "world") with "highest block is not at 0.5,253.5,0.5 (got '%location of {_highest}%')"
+
+	# lowest solid block
+	set block at location(0.5, 0.5, 0.5, "world") to air
+	set block at location(0.5, 1.5, 0.5, "world") to air
+	set block at location(0.5, 2.5, 0.5, "world") to dirt
```

Comment:
This will have issues with conflicting with other tests. In the future you'll be able to use `event-block` after one of my pulls, I forget which one.

Most tests revert the block back to what it was.

This one is problematic because it's near 0,0,0 either new location or I suggest making it restore

Reply:
I would imagine that a test should make no expectation regarding the state of the world - so if it is expecting a block to be something at a location, that should be explicitly set at the beginning of the test

Reply:
I think that's a good principle to abide by, but setting any blocks you set back to air seems like good defensive behavior in case a test isn't abiding by that.